Java/other/MergeSortedArrays.java

```diff
@@ -0,0 +1,76 @@
+import java.util.*;
+
+class MergeSortedArrays
+{
+	private static void merge(int arr1[], int arr2[])
+	{
+		// Iterate through all elements of arr2[] in reverse 
```

Comment:
@ritvij14 You can add a condition to check whether the array is null or not something like this
```
if (arr1 == null || arr2 == null) {
    return;
}
```

Reply:
Done.